### PR TITLE
Added type for createdDate and lastUpdatedDate

### PR DESCRIPTION
**Pull request by nico-shishkin**
# What changed

https://deploy-preview-1257--logz-docs.netlify.app/api/#operation/getAllDetailedTimeBasedAccount
----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/api/logzio-public-api.yml [hunk 1]
```diff
@@ -4414,15 +4414,15 @@ definitions:
...
         $ref: "#/definitions/Accessible"
       createdDate:
-        type: string
-        format: date-time
+        type: integer
+        format: int64
         description: >-
           Date this account was created.
           Format: `{yyyy}-{mm}-{dd}T{hh}:{mm}:{ss}Z`
         example: 2018-02-22T09:14:53Z
```
Comment (mirii1994) [inline _source/api/logzio-public-api.yml:4422]:
```suggestion
        example: 1627489797000
```

File: _source/api/logzio-public-api.yml [hunk 2]
```diff
@@ -4414,15 +4414,15 @@ definitions:
...
       lastUpdatedDate:
-        type: string
-        format: date-time
+        type: integer
+        format: int64
         description: >-
           Date this account was last updated.
           Format: `{yyyy}-{mm}-{dd}T{hh}:{mm}:{ss}Z`
```
Comment (mirii1994) [inline _source/api/logzio-public-api.yml:4428]:
```suggestion
          example: 1627489797000
```

**Review by mirii1994 — CHANGES_REQUESTED** [review]
Since we've changed the type, the description & example should be modified too

**Review by mirii1994 — APPROVED** [review]
Great! thank you!

**Review by YajB — APPROVED** [review]

**Pull request merged**